charts/reform-scan-blob-router/values.yaml

```diff
@@ -32,6 +31,7 @@ java:
     HANDLE_REJECTED_FILES_CRON: "0/30 * * * * *"
     TASK_SCAN_DELAY: "4000" # in millis
 
+    STORAGE_URL: https://reformscan.{{ .Values.global.environment }}.platform.hmcts.net
```

Comment:
just double checking, is prod env included in this template?

Reply:
Prod already overrides this setting: https://github.com/hmcts/cnp-flux-config/blob/master/k8s/prod/common/reform-scan/blob-router.yaml#L27

Reply:
cool beans 👍

Reply:
Deleted my already answered my comment by accident. Bringing it back:
> Prod already overrides this setting: https://github.com/hmcts/cnp-flux-config/blob/master/k8s/prod/common/reform-scan/blob-router.yaml#L27